.travis.yml

```diff
@@ -0,0 +1,5 @@
+sudo: false
+language: ruby
+rvm:
+  - 2.3.5
+before_install: gem install bundler -v 1.15.4
```

Comment:
Remove this file since we're no longer using Travis?

Reply:
Good catch, thanks! Didn't realize this was part of the bundler template